test/Microsoft.TestPlatform.Common.UnitTests/Utilities/MetadataReaderHelperTests.cs

```diff
@@ -22,7 +22,15 @@ public class MetadataReaderHelperTests
     public void MetadataReaderHelper_GetCollectorExtensionTypes()
     {
         string testAssetsPath = GetTestAssetsFolder();
-        var dataCollectorFilePath = Directory.GetFiles(testAssetsPath, "AttachmentProcessorDataCollector.dll", SearchOption.AllDirectories).Where(x => x.Contains("bin")).Single();
+        var dataCollectorFilePath =
+            Directory.GetFiles(testAssetsPath, "AttachmentProcessorDataCollector.dll", SearchOption.AllDirectories)
+            .Where(x => x.Contains("bin"))
+#if DEBUG
+            .Where(x => x.Contains("Debug"))
+#else
+            .Where(x => x.Contains("Release"))
+#endif
```

Comment:
Extract to static helper class or base class, and use `SomethingSomething.Configuration` in the filter? IntgrationTestBase does the same, but I don't think we have base classes for unit tests. Not sure, check other unit tests :)

Reply:
Done.